content/sensu-go/6.2/operations/control-access/oidc-auth.md

```diff
@@ -277,9 +277,16 @@ For example, if you have an Okta group `groups` and you set the `groups_prefix`
 
 #### Sensuctl login with OIDC
 
-1. Run `sensuctl login oidc`.
+Run `sensuctl configure` and select the OIDC authentication method.
 
-2. If you are using a desktop, a browser will open to `OIDC provider` and allow you to authenticate and log in.
+{{< code shell >}}
+$ sensuctl configure
+? Authentication method:
+  username/password
+❯ OIDC
+{{< /code >}}
+
+If you are using a desktop, a browser will open to `OIDC provider` and allow you to authenticate and log in.
 If a browser does not open, launch a browser to complete the login via your OIDC provider at following URL:
    - https://sensu-backend.example.com:8080/api/enterprise/authentication/v2/oidc/authorize
 
```

Comment:
If the sensuctl prompts for backend, namespace, and format continue after this, I should probably add a link back to the sensuctl setup page. But I'm not sure what happens after you select OIDC beyond this...

Reply:
For context, on line 280 we replaced `sensuctl login oidc` with `sensuctl configure`, however the first command will continue to work. The only advantage of using `sensuctl configure` with OIDC is that you don't need to authenticate first with a local user.

So to answer your question, once you select OIDC, we run the `sensuctl login oidc` command behind the scenes, so a browser will also open etc.